types/ember__component/test/lib/assert.ts

```diff
@@ -1,2 +1,5 @@
 /** Static assertion that `value` has type `T` */
 export declare function assertType<T>(value: T): T;
+
+// tslint:disable-next-line:no-unnecessary-generics
+export declare function assertExtendsType<P, T extends P = P>(value: T): T;
```

Comment:
Here, just use `// $ExpectType` – that should be good enough. Thanks.

Reply:
ooooh nice, I did not know about `$ExpectType` and very cool it can work without directly importing the types to be checked against